src/SourceBuild/content/eng/tools/BinaryToolKit/BinaryTool.CompareBinariesAgainstBaselines.cs

```diff
@@ -62,7 +62,7 @@ public partial class BinaryTool
     {
         var patterns = ParseBaselineFile(baselineFile);
 
-        if (mode == Modes.clean)
+        if (mode == Modes.Clean)
```

Comment:
It's confusing to me why you would only to do this for `Clean` but not `All` (which implicitly includes `Clean`).

Reply:
The reason is because `All` requires validation, which requires updating the baseline with patterns that are unused. It does not make sense to match all the patterns and then loop again to find all of the unused patterns.